POUF1.md

```diff
@@ -0,0 +1,448 @@
+* The metadata version will increment on an update to the metadata.
+* Root metadata must support [mappings of roles to lists of URLs as described in TAP 5](https://uptane.github.io/uptane-standard/uptane-standard.html#root_meta).
+* Repository mapping metadata will be included using a map file. The format of this file is described in {Formats}. In the mapping metadata, files are expressed using an enumerated list that supports wildcards.
+* If images are encrypted per ECU, metadata about encrypted images may be included in the custom field of targets metadata as described in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images). Targets metadata from the Director will still include metadata about unencrypted images so that this metadata can be compared with the metadata from the Image repository.
+* The custom field of targets metadata on the Director repository must support an [encrypted target field for encrypting images per ECU](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images). In addition, the custom field will contain a release counter and hardware id from both the Director and Image repositories. This POUF does not support the director adding a download URL to the custom field of targets metadata.
+* The custom field of targets metadata on the Director and Image repositories are not compared. This means that the must match functionality described in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images) is not supported. A client may choose to compare some fields of targets metadata, but these checks are not required by this POUF. If all other fields match, the custom metadata from the Director is used.
+* The Image repository’s targets metadata may include metadata about delegations. [Delegations may be any depth and may also be cyclic](https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta).
+* Snapshot metadata in this POUF does not include the root filename or version. These fields are included in implementations that do not use [TAP 5](https://github.com/theupdateframework/taps/blob/01726d203c9b9c029d26f6612069ce3180500d9a/tap5.md#downloading-metadata-and-target-files). Because this POUF implements TAP 5, the root filename and version do not need to be included in Snapshot metadata.
+
+# Usage
+Key management and image generation are not handled by this POUF. The POUF uses online keys for all roles to demonstrate Uptane functionality. This practice is not recommended for Uptane implementers.
+
+## Server Setup
+The Uptane Reference Implementation does the following setup before any updates can be installed:
+* The Director repository is initialized with the private keys for all roles with online keys. These keys are stored on the Director so they can be used to automatically sign valid metadata.
```

Comment:
Explain / rationalize

Reply:
o.o Director root definitely can't be an online key.  The demo's keys are all open, but I doubt that's what you mean?

Reply:
I added a sentence to clarify that not all keys should be stored online.